pkg/smokescreen/acl/v1/acl.go

```diff
@@ -0,0 +1,230 @@
+package acl
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Decider interface {
+	Decide(service, host string) (Decision, error)
+}
+
+type ACL struct {
+	Rules            map[string]Rule
+	DefaultRule      *Rule
+	GlobalDenyList   []string
+	GlobalAllowList  []string
+	DisabledPolicies []EnforcementPolicy
+	*logrus.Logger
+}
+
+type Rule struct {
+	Project     string
+	Policy      EnforcementPolicy
+	DomainGlobs []string
+}
+
+type Decision struct {
+	Reason  string
+	Default bool
+	Result  DecisionResult
+	Project string
+}
+
+func New(logger *logrus.Logger, loader Loader, disabledActions []string) (*ACL, error) {
+	acl, err := loader.Load()
+	if err != nil {
+		return nil, err
+	}
+
+	acl.DisablePolicies(disabledActions)
+
+	err = acl.Validate()
+	if err != nil {
+		return nil, err
+	}
+
+	acl.Logger = logger
+
+	if acl.DefaultRule == nil {
+		acl.Warn("no default rule set")
```

Comment:
What happens if no default rule is set? It might be worth including that in the output here

Reply:
If no default rule is set, services without a rule will be denied. ++ to updating the message!